### PR TITLE
Codegen class rename for Angular and AngularJS to reflect naming guidelines

### PR DESCRIPTION
**Pull request by damienpontifex**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: master for non-breaking changes and `3.0.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

see http://angularjs.blogspot.com.au/2017/01/branding-guidelines-for-angular-and.html for reason for naming change.

Previously TypescriptAngular referred to AngularJS and TypescriptAngular2 referred to Angular. This changes makes it clearer by using the names TypescriptAngularJs and TypescriptAngular to refer to AngularJS (1.x) and Angular (>2.x) 

Generator language flag was already following this convention so usage shouldn't change but class naming follows this convention now.

### CODE REVIEW

**Comment by wing328** [comment]
@damienpontifex thanks for the PR.

cc @Vrolijkx @taxpon @sebastianhaas @kenisteward @TiFu

**Comment by wing328** [comment]
@damienpontifex when you've time, I wonder if you can rebase this PR on the latest master to help resolve the merge conflicts.

**Comment by damienpontifex** [comment]
@wing328 done but build failures after rebase. Will have to spend a little longer resolving

File: modules/swagger-codegen/src/main/resources/META-INF/services/io.swagger.codegen.CodegenConfig [hunk 1]
```diff
@@ -67,9 +67,10 @@ io.swagger.codegen.languages.Swift4Codegen
 io.swagger.codegen.languages.Swift3Codegen
 io.swagger.codegen.languages.SwiftCodegen
 io.swagger.codegen.languages.TizenClientCodegen
-io.swagger.codegen.languages.TypeScriptAngular2ClientCodegen
 io.swagger.codegen.languages.TypeScriptAngularClientCodegen
 io.swagger.codegen.languages.TypeScriptAureliaClientCodegen
+io.swagger.codegen.languages.TypeScriptAngularJsClientCodegen
+>>>>>>> Rrename for angular and angularjs to reflect naming guidelines
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/META-INF/services/io.swagger.codegen.CodegenConfig]: @damienpontifex please remove this line.

**Comment by damienpontifex** [comment]
@wing328 I think that's fixed it all. I can squash the changes into one commit when/If merged.

**Comment by damienpontifex** [comment]
@wing328 any further thoughts/feedback on this?

**Comment by wing328** [comment]
@damienpontifex last time I checked, this PR is good to merge but I want to merge https://github.com/swagger-api/swagger-codegen/pull/6130 first

**Comment by wing328** [comment]
#6310 has been merged into master.

Will merge this one over the weekend.

**Comment by wing328** [comment]
@damienpontifex thanks for the PR, which has been merged into master. I've made the following minor changes:

d8fceda - update TS Angular-related Windows batch files
ccfe7f5 - minor fix to TS AngularJS bin script

**Pull request merged**